### PR TITLE
Adds security best practice doc.

### PR DESCRIPTION
**Pull request by madjam**

### CODE REVIEW

File: docs/faq/index.md [hunk 1]
```diff
@@ -62,3 +65,4 @@ We track bugs and new feature requests in the MXNet Github repo in the issues fo
 
 ## Roadmap
 MXNet is evolving fast. To see what's next and what we are working on internally, go to the [MXNet Roadmap](https://github.com/dmlc/mxnet/labels/Roadmap).
+I
```
Comment (szha) [inline docs/faq/index.md]: Keyboard misfire?

Comment (madjam) [reply docs/faq/index.md]: Oops. Fixed it.

**Comment by madjam** [comment]
I would like this to be included in 1.0.
Please review for accuracy and completeness.
@mli @piiswrong @nswamy

**Comment by szha** [comment]
Please do a rebase onto master and rerun tests. PRs cannot be merged without successful tests right now.

**Pull request closed without merging**